packages/editor/src/store/utils/notice-builder.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { SAVE_POST_NOTICE_ID, TRASH_POST_NOTICE_ID } from '../constants';
+
+/**
+ * External dependencies
+ */
+import { get, includes } from 'lodash';
+
+/**
+ * Builds the arguments for a success notification dispatch.
+ *
+ * @param {Object} data Incoming data to build the arguments from.
+ * @return {Array}  Arguments for dispatch. An empty array signals no
+ * notification should be sent.
```

Comment:
By convention, I'd recommend aligning all but the first lines with the start of the description from the previous line. This isn't documented in the JavaScript Documentation Standards, but is mentioned in passing in the PHP standards:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays

>(aligned with Description, if wraps to a new line)

Reply:
The [WP standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions) you linked to earlier have descriptions aligned _per group_.  So for instance the @params have their descriptions aligned, and the @return is not necessarily aligned with the previous group (at least from the example given).  So that's what I followed in my pass through the doc blocks.

Reply:
> The [WP standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions) you linked to earlier have descriptions aligned _per group_. So for instance the @params have their descriptions aligned, and the @return is not necessarily aligned with the previous group (at least from the example given)

Yes, this sounds right. My point was toward the `@return` specifically aligning with itself. In your revisions, I think there are issues with mixed whitespace / tabbing.

![image](https://user-images.githubusercontent.com/1779930/53515134-4a262f80-3a97-11e9-9c5c-a68a1b40d631.png)

Reply:
argh, looks fine in the IDE.  I *hate* when the IDE doesn't cooperate with documentation requirements.  I'll revisit but likely won't get to it today.